utils/node-startup-prep/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:alpine AS builder
```

Comment:
alpine? If you will use on debian you better use debian, even if the binary is statically compiled.

Reply:
I think golang crosscompiles and has no links to any C library, so no worries here.